lib/page/prompt.js

```diff
@@ -55,6 +55,11 @@ docReady(() => {
 		document.querySelector('#label').textContent = promptOptions.label;
 	}
 
+	if (promptOptions.buttonLabels) {
+		document.querySelector('#ok').textContent = promptOptions.buttonLabels.ok;
```

Comment:
There should be a check on each property being defined too, so you don't have to define both labels